### DISCUSSION TITLE
Potential friction improvements for Developer Blog

### DISCUSSION THREAD

**Discussion by perkinsjr** [remix-run/remix#1464] [General]
Hey everyone / @kentcdodds 

I just spent some time using the Developer blog guide on stream, great guide over all but here are a few things I thought about when running through it:

1. In the  `your-first-route section` it states when adding the Link.

> You can put it anywhere you like, you might want to just delete everything that's there.

If you remove everything inside the body, this causes the app to of course only display the Link and never render the rest of the Remix app. If this is the first time someone has played with Remix , they maybe be confused.

2. The project uses Typescript which is stated at the top, for users not using Typescript it might be nice to have callouts that state "Typescript Only", for when you update Types. So JS users can skip over and not get lost in the Types .  

3. It would be nice to reference out to things like `<Outlet/>`  or `useActionData` for example. So, a user could investigate the use case as they go through the quickstart. 

4. Add the example application so people can look at the GitHub version in case they make some mistakes.

**Comment by kentcdodds** [comment]
Thanks for this feedback @perkinsjr!

1. Good point, would you be interested in making a PR to make this improvement?
2. I think I'm ok with non-TS users having to read through the TS stuff. I want them to see what it would be like with TypeScript even if they're not using it yet.
3. That's a good point. When we talk about them we should link to the docs for those things. A PR is welcome here.
4. We could add a link to [the finished version](https://github.com/remix-run/remix/tree/main/examples/blog-tutorial) at the beginning. Again, a PR would be appreciated :)

Thanks again for caring enough to take the time to bring this up!

**Reply by perkinsjr** [reply]
Absolutely, I will draft up a PR for the changes. 

👍
